### DISCUSSION TITLE
人工智能：朋友還是敵人？

### DISCUSSION THREAD

**Discussion by titan92601** [1101-GS3073-C/Python-Community-GS3073-C#573] [科技新知]
是敵？是友？
O NO GA DAM
https://www.aljazeera.com/podcasts/2023/1/4/artificial-intelligence-friend-or-foe

**Comment by number32984** [comment]
是敵人也是朋友，看你怎們使用他

**Comment by awehao** [comment]
亦敵亦友，對我有害的一律剷除剷除

**Comment by Aerofe** [comment]
只要人給他附加限制的話，AI對我們的幫助肯定很大！

**Comment by magicmicheal** [comment]
是敵是友這個問題，有很大的一部份取決於使用的人吧

**Comment by bww0401** [comment]
我認為科技始終來自於人性，最主要還是看我們怎麼去運用吧

**Comment by larry40305** [comment]
總有一天會變敵人

**Comment by boson13579** [comment]
要看用他的人是抱持惡意還是善意。例如剛剛看到一篇文章是關於有犯罪份子用AI詐騙，有人卻開發出AI警察幫助警察破案，所以到底AI是敵是友不能以概論全。

**Comment by Valentinaa0214** [comment]
我認為都不是，它只是一種工具。我們使用它的方式對我們自身產生的影響有好有壞。
它可以被用來做任何事，包括對我們有利和不利的。但若使用得當，AI可以幫助我們解決各種問題，提高生產效率和生活品質。
